pkg/statistics/handle/autoanalyze/priorityqueue/calculator.go

```diff
@@ -56,6 +56,13 @@ func (pc *PriorityCalculator) CalculateWeight(job AnalysisJob) float64 {
 	// when we apply the log10 function, the resulting value is more meaningful and reasonable.
 	indicators := job.GetIndicators()
 	changeRatio := 100 * indicators.ChangePercentage
+	if j, ok := job.(*DynamicPartitionedTableAnalysisJob); ok {
```

Comment:
This is so hack. I think we should consider introducing a new job for the global index instead of lacking this information here.

Reply:
Or at least keep it in the refresher.